configure.ml

```diff
@@ -717,20 +723,14 @@ let get_lablgtkdir () =
 
 let check_lablgtk_version () =
   let v, _ = tryrun camlexec.find ["query"; "-format"; "%v"; "lablgtk3"] in
-  (true, v)
-
-(* ejgallego: we wait to do version checks until an official release is out *)
-(*  try
-    let vi = numeric_prefix_list v in
-    (* Temporary hack *)
-    if vi = ["3";"0";"beta3"] then (false, v) else
-    let vi = List.map s2i vi in
-    if vi < [3; 0; 0] then
+  try
+    let vl = numeric_prefix_list v in
+    let vn = List.map int_of_string vl in
```

Comment:
This kind of code is very fragile if the version does contain strings; but it is not easy to fix as string comparison will fail. Only solution is to do the cleanup and vendor opam's code for version comparison.